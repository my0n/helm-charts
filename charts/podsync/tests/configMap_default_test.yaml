suite: test configMap
templates:
  - configMap.yaml
tests:
  - it: should match snapshot
    asserts:
      - matchSnapshot: {}