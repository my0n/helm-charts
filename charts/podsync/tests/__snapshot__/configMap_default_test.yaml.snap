should match snapshot:
  1: |
    apiVersion: v1
    data:
      config.toml: |
        # template
    kind: ConfigMap
    metadata:
      name: RELEASE-NAME-podsync
